.travis.yml

```diff
@@ -0,0 +1,44 @@
+sudo: required
+
+dist: trusty
+
+language: generic
+
+python:
+  - 2.7
+
+compiler:
+  - gcc
+  - clang
+
+env:
+  matrix:
+    - ROS_DISTRO=indigo
+
+# command to install dependencies
+install:
+  - sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu trusty main" > /etc/apt/sources.list.d/ros-latest.list'
+  - wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
+  - sudo apt-get update -qq
+  - sudo apt-get install python-rosdep python-catkin-tools -qq
+  - sudo rosdep init
+  - rosdep update
+  # Make a workspace
+  - mkdir -p /tmp/image_pipeline_ws/src
+  # Add this folder to the workspace
+  - ln -s $(pwd) /tmp/image_pipeline_ws/src/image_pipeline
+  # Install dependencies
+  - cd /tmp/image_pipeline_ws/src
+  - git clone https://github.com/ros-perception/vision_opencv.git
+  - (cd vision_opencv && git checkout 7ff3fa72ac05377b9d7c8511ef9378f0e50eff82)
```

Comment:
actually, the build fails with this change. https://github.com/ros-perception/vision_opencv/commit/2be3371fa6f793b9562f0f891fe561effd511ca9

by saying 
```
[cv_bridge] In file included from /home/wkentaro/ros/indigo/src/ros-perception/vision_opencv/cv_bridge/src/cv_bridge.cpp:36:0:                                 
[cv_bridge] /home/wkentaro/ros/indigo/src/ros-perception/vision_opencv/cv_bridge/src/boost/endian/conversion.hpp:12:47: fatal error: boost/predef/detail/endian_compat.h: No such file or directory 
[cv_bridge]  #include <boost/predef/detail/endian_compat.h>                                                                                                    
[cv_bridge]                                                ^                                                                                                   
[cv_bridge] compilation terminated.                                                                                                                            
[cv_bridge] make[2]: *** [src/CMakeFiles/cv_bridge.dir/cv_bridge.cpp.o] Error 1      
```